crates/solver/src/driver.rs

```diff
@@ -385,13 +385,15 @@ impl Driver {
             current_block::block_number(&self.block_stream.borrow())?;
 
         let before_count = auction.orders.len();
-        self.in_flight_orders.update_and_filter(&mut auction);
+        let inflight_order_uids = self.in_flight_orders.update_and_filter(&mut auction);
         if before_count != auction.orders.len() {
             tracing::debug!(
-                "reduced {} orders to {} because in flight at last seen block {}",
+                "reduced {} orders to {} because in flight at last seen block {}, orders in flight: {:?}",
                 before_count,
                 auction.orders.len(),
-                auction.block
+                auction.latest_settlement_block,
+                inflight_order_uids
+
```

Comment:
extraneous newline. `rustfmt` doesn't pick it up because its super conservative with formatting `macro!` invocations.

Reply:
```suggestion
```